crates/indexer/src/accountsdb/accounts/auction_cache.rs

```diff
@@ -0,0 +1,127 @@
+use indexer_core::{
+    db::{
+        insert_into,
+        models::{AuctionCache, ListingMetadata},
+        tables::{auction_caches, listing_metadatas},
+    },
+    pubkeys::find_auction_data_extended,
+};
+use metaplex::state::AuctionCache as AuctionCacheAccount;
+
+use crate::{prelude::*, Client};
+
+pub(crate) async fn process(
+    client: &Client,
+    cache_key: Pubkey,
+    cache: AuctionCacheAccount,
+) -> Result<()> {
+    // let mut acct = client
+    //     .get_account(&keys.cache)
+    //     .context("Failed to get auction cache")?;
+
+    // let cache = AuctionCacheAccount::from_account_info(&util::account_as_info(
+    //     &keys.cache,
+    //     false,
+    //     false,
+    //     &mut acct,
+    // ))
+    // .context("Failed to parse AuctionCache")?;
+
+    let AuctionCacheAccount {
+        metadata,
+        auction,
+        vault,
+        timestamp,
+        store,
+        auction_manager,
+        ..
+    } = cache;
+
+    // TODO: store listing metadata
```

Comment:
This commented out code good to remove now?

Reply:
I believe so.  That's up next on my list, since auction caches still don't technically work yet.